README.md

```diff
@@ -1,2 +1,170 @@
 # orchestrator-helm-chart
-Helm chart to deploy the Orchestrator solution suite, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator,  Knative Eventing, Knative Serving, Data Index and Job Service.
+Helm chart to deploy the Orchestrator solution suite on OpenShift, including Janus IDP backstage, SonataFlow Operator, OpenShift Serverless Operator, Knative Eventing, Knative Serving, Data Index and Job Service.
+
+This chart will deploy the following on the target OpenShift cluster:
+  - Janus IDP backstage
+  - SonataFlow Operator
+  - OpenShift Serverless Operator
+  - Knative Eventing
+  - Knative Serving
+  - Data Index Service
+  - Job Service
+  - PostgreSQL Databases 
+  - Sample workflow (event-timeout)
+
+## Installation
+Build helm dependency:
+```console
+$ git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
+$ cd orchestrator-helm-chart/charts
+$ helm dep update orchestrator
+```
+Perform a first pass install:
+```console
+$ helm install orchestrator orchestrator
+NAME: orchestrator
+LAST DEPLOYED: Thu Nov  2 09:02:29 2023
+NAMESPACE: jianrong
+STATUS: deployed
+REVISION: 1
+```
+Wait for about 1 minute, and then check to make sure the SonataFlow operator and OpenShift Serverless operator have started.
```

Comment:
Can you include the commands recommended to the user in the output of the helm command in a similar fashion as done here:
https://github.com/parodos-dev/serverless-workflow-examples/blob/main/escalation-eda/helm/eda-infra/templates/NOTES.txt

Reply:
Done!